pkg/healthcheck/healthcheck.go

```diff
@@ -873,16 +880,34 @@ func validateControlPlanePods(pods []corev1.Pod) error {
 	}
```

Comment:
While we're at it, I think we can move "proxy-injector" into the slice above now that it's a required component.

Reply:
(:point_up: this was in reference to the `names` slice in `validateControlPlanePods()`)

Reply:
hmm, i intentionally left it as-is because i didn't want newer CLIs to hang when checking an older control-plane that doesn't have a `proxy-injector`. let me know what you think.

Reply:
That's a good point I didn't consider. Let's leave it as is for some more time then :+1: